pandas/tests/indexes/datetimes/test_missing.py

```diff
@@ -48,3 +48,11 @@ def test_fillna_datetime64(self):
                             pd.Timestamp('2011-01-01 11:00', tz=tz)],
                            dtype=object)
             self.assert_index_equal(idx.fillna('x'), exp)
+
+    def test_fillna_timezone(self):
```

Comment:
can you move to ``pandas/tests/frames/test_missing.py``; also add a test in ``series/test_missing.py`` (if not already there).

can you also test with ``backfill`` (for both)